travis.sh

```diff
@@ -26,6 +26,9 @@ export TARGET_REPO_NAME=${TRAVIS_REPO_SLUG##*/}
 export PYTHONUNBUFFERED=${PYTHONUNBUFFERED:1}
 export _FOLDING_TYPE=travis
 
+# Update libseccomp to allow statx syscalls
+(sudo apt update && sudo apt install -y libseccomp2) &> "$(mktemp)"
```

Comment:
Why do you forward it to a temporary file?
And why do yu need to to update first?

Reply:
Without updating libseccomp2 will be at 2.3.2, which will block statx syscall.

Reply:
I meant the "apt-get update" :)

Reply:
Just read that Travis does not update automatically (any more?)